sycl/include/CL/sycl/detail/pi.h

```diff
@@ -246,8 +247,17 @@ static const uint8_t PI_DEVICE_BINARY_OFFLOAD_KIND_SYCL = 4;
 /// pi_device_binary_struct.DeviceTargetSpec
 ///
 #define PI_DEVICE_BINARY_TARGET_UNKNOWN "<unknown>"
+/// SPIR: standard portable IR for OpenCL 32-bit version
 #define PI_DEVICE_BINARY_TARGET_SPIRV32 "spir"
-#define PI_DEVICE_BINARY_TARGET_SPIRV64 "spir64";
+/// SPIR: standard portable IR for OpenCL 64-bit version
```

Comment:
There is some confusion, AFAIK we use **SPIR-V** binaries, not **SPIR**, this is not same things. I think this should be aligned.

Reply:
Yep, I see the confusion here.
This comment was initially taken from `llvm::Triple` class header. In a way, this comment section describes the target triples' rationale and not the images' specifics. I'll come up with a way to reword this so that **SPIR-V** binaries usage is made clear.

Reply:
I think I should remove the 'subarchs' comments from here altogether, while updating the comments around the usages of these aliases, clarifying that these images were compiled from a SPIR-V binary.

Reply:
Done as described in the previous comments. Will this be sufficient?

Reply:
I'm still confused with defining PI_DEVICE_BINARY_TARGET_**SPIRV**32 with **"spir"** .

Reply:
I can think of the following rationale here:

`PI_DEVICE_BINARY_TARGET_SPIRV32` represents the requirements for the image, implying "the image must be SPIR-V, 32-bit".
`spir` actually stands for an LLVM device target triple. This construct expands well beyond AOT compilation for SYCL devices.

Part of the `#define`s' job is to map image requirements onto the target architectures. It's kinda like: "We have a device which is described by triple `spir-unknown-linux-sycldevice`. We'd like to load SPIR-V 32 bit images into it"

Reply:
Could you please add a comment to clarify this?

Reply:
Sure, especially since I had to figure it out myself for quite a while :)

Reply:
I've updated this comment section, as well as the one in `pi_device_binary_struct`